lib/core/settings/host-settings.component.html

```diff
@@ -128,6 +128,13 @@ <h3>{{'CORE.HOST_SETTINGS.TITLE' | translate}}</h3>
                         <input id="logout-url" matInput placeholder="{{ 'CORE.HOST_SETTINGS.REDIRECT_LOGOUT'| translate }}"
                                name="redirectUriLogout" formControlName="redirectUriLogout">
                     </mat-form-field>
+
+                    <mat-form-field class="adf-full-width" floatLabel="Public Urls">
+                        <mat-label>{{ 'CORE.HOST_SETTINGS.PUBLIC_URLS'| translate }}</mat-label>
+                        <input id="public-url" matInput placeholder="{{ 'CORE.HOST_SETTINGS.PUBLIC_URLS'| translate }}"
```

Comment:
I noticed that after editing the publicUrls from the settings page, the new value is stored as an atring, not array as expected. Please check, @eromano

Reply:
I guess we should add a check inside the js-api that the publicUrls is array

Reply:
I have added a PR with isArray check https://github.com/Alfresco/alfresco-js-api/pull/423 in order to speed things up, in case you agree with this proposal @eromano please check

Reply:
removing the input field for publicUrls from settings template is a quick workaround for demo-shell...

Reply:
is not a problem that is a string after when it gets parsed it becomes an array from what I checked